step.yml

```diff
@@ -226,3 +234,15 @@ outputs:
 
         - $BITRISE_DEPLOY_DIR/ios_app.ipa=>https://ios_app/public/install/page
         - $BITRISE_DEPLOY_DIR/android_app.apk=>https://android_app/public/install/page|$BITRISE_DEPLOY_DIR/ios_app.ipa=>https://ios_app/public/install/page
+  - BITRISE_PERMANENT_DOWNLOAD_URL_MAP:
```

Comment:
Mig it would wort to update `BITRISE_PUBLIC_INSTALL_PAGE_URL_MAP` description too with `public_install_page_url_map_format`

Reply:
👍